source/includes/v2/_brokerage-api-withdrawals.md

```diff
@@ -0,0 +1,174 @@
+`document.properties` | `document.properties`        | *map<string, string>*                              | *body* | dictionary of key-value pairs with custom properties
+`document.assetid` | `document.asset_id`          | *int64*                                            | *body* | ID of the asset to use
+`document.brokerAccountId` | `document.broker_account_id` | *int64*                                            | *body* | ID of the broker account to use
+`document.destinationDetails` | `document.destination_details` | *[DestinationDetails](#destinationdetails-object)* | *body* | destination parameters of the withdrawal (address and tag info)
+`document.Amount` | `document.amount` | *[DestinationDetails](#destinationdetails-object)* | *body* | destination parameters of the withdrawal (address and tag info)
+`signature` | `signature`                  | *optional*, *string*                               | *body* | Base64-encoded RSA signature of the document signed with the Customer's private key. The exact format of the document to be signed is described below.
+
+Withdrawal document format (order of parameters is important):
+```
+['36585f34-d311-433a-87f1-1751b08480c3','300578305','100000140','0x663e933ECdc5b1acbaCB87F4aa1636cd05837613','null','null','0.1','UserId:test','WalletId:12345']
+```
+
+Parameters used:
+0: IdempotencyId `36585f34-d311-433a-87f1-1751b08480c3`
+1: AssetId `300578305` (BNB)
+2: BrokerAccountId `100000140`
+3: DestinationAddress `0x663e933ECdc5b1acbaCB87F4aa1636cd05837613`
+4: DestinationTag `null`
+5: DestinationTagType `null`
+6: Amount: `0.1`
+7: Properties:
+7.1: UserId: `test` (corresponds to UserNativeId)
+7.2: WalletId: `12345` (corresponds to AccountReferenceId)
+
+Order of Properties sent from the client side is not important. However, it is important to sort them alphabetically by keys during document creation. Received pairs of properties will be sorted during signature validation internally.
+ 
+The presented document format is used whenever client wishes to execute signed withdrawal. Withdrawal document has to constructed on the client side and signed with customer private key.
+
+Notes on the document format:
+- Please use string literal `null` for parameters without value
+- Please format numeric values with at least one fractional digit, even if the value is an integer (i.e. 2 has to become 2.0)
+
+### Query Parameters
+
+### Response
+
+```json
+POST /api/v2/withdrawals
+
+> Response: 200 (application/json) - success response
+
+{
+ "id":106000003
+}
+```
+
+```protobuf
+swisschain.sirius.api.withdrawals.Withdrawals.Execute
+
+> Response: (application/grpc) - success response
+
+message WithdrawalExecutePayload {
+  int64 id = 1;
+}
+
+message WithdrawalExecuteError {
+  enum WithdrawalExecutionErrorCode {
+    UNKNOWN = 0;
+    INVALID_PARAMETERS = 1;
+    DOMAIN_PROBLEM = 2;
+    TECHNICAL_PROBLEMS = 3;
```

Comment:
According to https://github.com/swisschain/Sirius.Api#endpoint-responses it should be `TECHNICAL_PROBLEM` (the same issue on the implementation).

Reply:
`+` Fixed for Sirius.Api too